### ISSUE TITLE
Footer navlinks can be confusing

### ISSUE THREAD

**Issue by JackDunnNZ** [JuliaDocs/Documenter.jl#1347]
This is a very minor comment, but a couple of times I have gotten confused when navigating through docs using the navigation links at the bottom, because the name of the next page that is shown only uses the page title and doesn't show information about the hierarchy of the site pages.

An example is from the Plots.jl docs where on [this page](https://docs.juliaplots.org/latest/output/) the next page is called "Overview", but really we are descending into the "Attributes" group where the first page is "Overview". It can confusing to just see "Overview" as there's no context about what this is an overview of.

Another thing is that there could be name repetition, e.g. with the following page structure
```
- Home
- Page1
- Section1
    - Home
    - Page1
    - Page2
...
```

If I am on the top-level "Page1", the navigation links at the bottom will both show "Home", even though the next page is really "Section1 - Home".

**Comment by mortenpi** [comment]
Writing the whole hierarchy out has the downside that the link may get pretty long, and that can cause issues with the layout. But yes, in principle, showing more information there is a good idea.

One partial fix would be to add `title` attributes that contain the whole hierarchy.

**Comment by JackDunnNZ** [comment]
Yes, I think it would be hard to justify showing the full information everytime. I'm not sure what the best fix might be, just wanted to mention the problem in case there is a nice solution.

**Comment by zdroid** [comment]
One approach may be to use "Section / Page" format (as it is in header) instead of just "Page" (as it is now). How does that suit you @JackDunnNZ?

**Comment by JackDunnNZ** [comment]
I think that would deal with this problem and also provide a good
indication when moving between sections. I would not want it to be too
noisy by default just to deal with this. A small variation to consider
could be to use "Section / Page" when the next page is from a different
section, but just use "Page" when the section is the same between the pages?
